cmd/telemetrygen/internal/logs/logs.go

```diff
@@ -53,8 +52,14 @@ func Start(cfg *Config) error {
 		return fmt.Errorf("'telemetrygen logs' only supports insecure gRPC")
 	}
 
+	endpoint := cfg.Endpoint
+	if host, _, err := net.SplitHostPort(endpoint); err == nil {
+		if u, err := url.Parse(host); err == nil && u.Scheme == "" {
+			endpoint = fmt.Sprintf("passthrough:///%s", endpoint)
+		}
+	}
```

Comment:
IMO if we do this it should happen at `Config` construction time instead of in the `Start` function

Reply:
I am not sure when that is? We parse config from args and pass it to the command iirc. I can take a look, but I'd argue that this change is an internal change to how we handle the input in some specific circumstances to make the gRPC lib a bit happier, not a config manipulation. If we dumped that value to a customer, I'd expect to hear questions about that passthrough scheme when it's really not relevant to the user.

Reply:
> I'd argue that this change is an internal change to how we handle the input in some specific circumstances to make the gRPC lib a bit happier, not a config manipulation. If we dumped that value to a customer, I'd expect to hear questions about that passthrough scheme when it's really not relevant to the user.

That's a good point, you have convinced me about not doing this when constructing `Config`.

I still think we should abstract this logic to a separate `func gRPCEndpoint(string) string` helper or a `Config` method, since we are using this for all signals. What do you think?

Reply:
sure, I can do that